week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productprice) {
+
+  return productprice+(productprice*20/100);
```

Comment:
we only need to calculate SalesTax out of Product Price.
---------- 100%   <-- if this product price
-- 20%                 <-- this is what makes 20% of it
But if we return productprice + 20% of it, it will be:
------------ 120%  <-- this much
but function calculateSalesTax need only to return calculated 20% Tax.